source/lib.ts

```diff
@@ -144,13 +144,24 @@ const parseOptions = (passedOptions: Partial<Options>): Configuration => {
 
 			return request.ip
 		},
-		handler(
+		async handler(
 			_request: Request,
 			response: Response,
 			_next: NextFunction,
 			_optionsUsed: Options,
-		): void {
-			response.status(config.statusCode).send(config.message)
+		): Promise<void> {
+			let { message } = config as { message: unknown }
```

Comment:
Instead of doing this, could you please change the type of `message` in the `types.ts` file [here](https://github.com/nfriedly/express-rate-limit/blob/master/source/types.ts#L189) to `any | ValueDeterminingMiddleware<any>`?